server/sonar-server/src/main/resources/org/sonar/server/computation/ws/quality_gate_status-example.json

```diff
@@ -0,0 +1,70 @@
+{
+  "qualityGateDetails": {
+    "level": "ERROR",
+    "conditions": [
+      {
+        "metric": "new_coverage",
+        "op": "LT",
+        "period": "1",
+        "warning": "",
```

Comment:
I understand your point, once again I show the data to PO and it was considered acceptable